public/toolkit/styles/molecules/_react-tables.scss

```diff
@@ -459,7 +460,7 @@ $narrow-column-width: 120;
   }
 
   .td-offset-right {
```

Comment:
I'm a little confused at why this class should have the substatus width applied, seems like the class is more generic than just substatus?

Reply:
🤷 it's really not that generic, it's for the column to the right of the status column in the lease-up applications table. 
whole thing needs to be overhauled, idea was just that this is a quick fix before we redo the styling. 

Maybe it'd help if I just rename the class?